test/postgresql-persistent-template.json

```diff
@@ -0,0 +1 @@
+../examples/postgresql-persistent-template.json
```

Comment:
Why creating symlinks? Isn't it possible to use the templates directly from `examples` directory? So there is not that much noise in the repo root?

Reply:
Good point. I need to clarify it If it works.

Reply:
It works, but during the syncing sources from upstream -> downstream, double symlinks are not allowed and there there is this symlink.

Reply:
What do you mean? `examples/postgresql-*-template.json` do not seem to be symlinks.

Reply:
You are right, basaed on the dist-git repo . I will fix it.

Reply:
This should be fixed by https://github.com/sclorg/postgresql-container/pull/499/commits/29f518a1fe81fd8650b3600c546f6f7f3af91111